include/grpc/grpc_security.h

```diff
@@ -834,30 +838,73 @@ typedef void (*grpc_tls_on_credential_reload_done_cb)(
     grpc_tls_credential_reload_arg* arg);
 
 /** A struct containing all information necessary to schedule/cancel a
-    credential reload request.
-    - cb and cb_user_data represent a gRPC-provided
-      callback and an argument passed to it.
-    - key_materials_config is an in/output parameter containing currently
-      used/newly reloaded credentials. If credential reload does not result in
-      a new credential, key_materials_config should not be modified. The same
-      key_materials_config object can be updated if new key materials is
-      available.
-    - status and error_details are used to hold information about
-      errors occurred when a credential reload request is scheduled/cancelled.
-    - config is a pointer to the unique grpc_tls_credential_reload_config
-      instance that this argument corresponds to.
-    - context is a pointer to a wrapped language implementation of this
-      grpc_tls_credential_reload_arg instance.
-    - destroy_context is a pointer to a caller-provided method that cleans
-      up any data associated with the context pointer.
+    credential reload request. To help understand each fields, we mark each
+    field with labels. Here are their explanations
+    -------------------------label explanation----------------------------------
+    [USER-MANAGED]: fields that are managed directly by the end users
+    [SYSTEM-PROVIDED]: fields that needed to retrieve implementations in C core
+    and would be provided by the gRPC stack
+    [WRAP-LANG]: fields that needed to retrieve the particular implementation in
+    wrap language
+    [ASYNC_ONLY]: fields that only used when in asynchronous mode. If a field
+    doesn't have this label, we can assume it should be applied to both
+    synchronous and asynchronous mode.
+    Note that if a field is marked both as [USER-MANAGED] and [SYSTEM-PROVIDED],
+    it indicates that the field is initiated by the gRPC stack, but users
+    need to properly manage them to correctly perform credential reloading.
+    ----------------------label explanation ends--------------------------------
+    - cb[USER-MANAGED][ASYNC_ONLY]: a callback-function provided for users to
```

Comment:
This field is provided by gRPC stack not users.

Reply:
you can say "a callback-function provided by the gRPC stack for users to call to ...:"